src/server/bin/kibana.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
```

Comment:
Looks like the permissions on this file have changes from 755 to 644. Intentional? Doesn't that mean it's not executable anymore?